### PR TITLE
Add urbansim_templates

### PR DESCRIPTION
**Pull request by smmaurer**
Creating a recipe for urbansim_templates v0.1.1

https://github.com/udst/urbansim_templates

### CODE REVIEW

**Comment by smmaurer** [comment]
@conda-forge/help-python Hi, i think this is ready for review!

**Comment by beckermr** [comment]
LGTM! Thanks for the contribution!

**Comment by smmaurer** [comment]
@beckermr Thanks!!

File: recipes/urbansim_templates/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    - pip
+  run:
+    - choicemodels >= 0.2
+    - numpy >= 1.14
+    - orca >= 1.4
+    - pandas >= 0.22
+    - patsy >= 0.4
+    - python
+    - statsmodels >= 0.8
+    - urbansim >= 3.1
```
Comment (jakirkham) [inline recipes/urbansim_templates/meta.yaml]: The syntax for these is unfortunately incorrect. There should be no space after the `>=` (only before). So it should be `urbansim >=3.1`. This applies to the rest as well. Mentioning this as the recipe is [failing to convert]( https://travis-ci.org/conda-forge/staged-recipes/builds/489687358 ) due to this syntax error. Could you please send a PR to fix the recipe, @smmaurer?

Comment (smmaurer) [reply recipes/urbansim_templates/meta.yaml]: @jakirkham Sorry, thanks so much for pointing this out! I just opened a new PR (#7734) that fixes the syntax.

**Pull request merged**